docs/gendocs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# just run all of the scripts in default mode in the right order
```

Comment:
I'm a little sad that you didn't move and reuse my existing script https://github.com/DOAJ/doaj/blob/develop/run_docs.sh

Reply:
haha, oops, I'm sorry I didn't spot that file existed!